docs/clients/grpc/projections.md

```diff
@@ -76,6 +79,11 @@ Projection management operations are exposed through a dedicated client.
 Sample available soon
 ```
 
+</xode-block>
+<xode-block title="Go" code="connectionString">
```

Comment:
needs a line feed to render correctly
this while block needs to be befor the java one